core-feature-pack/src/main/resources/content/bin/domain.conf.ps1

```diff
@@ -25,33 +25,34 @@ if (-Not(test-path env:JBOSS_MODULES_SYSTEM_PKGS )) {
   $JBOSS_MODULES_SYSTEM_PKGS="org.jboss.byteman"
 }
 
+# Set default values if none have been set by the user
```

Comment:
How is that change related to the `-server` unification?

Reply:
The change to standalone.conf.ps1 does not change the handling of JAVA_OPTS
by standalone.ps1.  Function Get-Java-Arguments in common.ps1 calls function
Get-Java-Opts.  This function checks the setting of PRESERVE_JAVA_OPTS and
JAVA_OPTS.  It chooses the user's JAVA_OPTS if set or the values set in
standalone.conf.ps1.